cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/ZabbixWidget.kt

```diff
@@ -0,0 +1,225 @@
+package com.cognifide.cogboard.widget.type
+
+import com.cognifide.cogboard.CogboardConstants
+import com.cognifide.cogboard.config.service.BoardsConfigService
+import com.cognifide.cogboard.widget.AsyncWidget
+import com.cognifide.cogboard.widget.Widget
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import io.vertx.core.logging.Logger
+import io.vertx.core.logging.LoggerFactory
+import kotlin.math.pow
+
+class ZabbixWidget(
+        vertx: Vertx,
+        config: JsonObject,
+        private val boardServ: BoardsConfigService = BoardsConfigService()
+) : AsyncWidget(vertx, config, boardServ) {
+
+    private val selectedMetric: String = config.getString(METRIC, "")
+    private val host: String = config.getString(HOST, "")
+    private val maxValue: Int = config.getInteger(MAX_VALUE, 0)
+    private val range: JsonArray = config.getJsonArray(RANGE, JsonArray())
+
+    val logger: Logger
+        get() = LoggerFactory.getLogger(ZabbixWidget::class.java)
```

Comment:
Make this variable a `companion` object so it is the same instance for every class instance

Reply:
done